WasteManagement/WasteContainer/doc/spec.md

```diff
@@ -33,6 +33,19 @@ to keep backwards compatibility with current FIWARE reference implementations.
     + Allowed values: Interval [0,1]. 
     + Optional
     
++ `weight` : Weight of the container load.
+    + Attribute type: [Number](http://schema.org/Number)
```

Comment:
as per latest discussions I propose the following

name the attribute as `cargoWeight` , to denote precisely that we are talking about the weight of the container's cargo. 

We need as well to link the TimeInstant metadata to its description, please check other attributes. 

BTW we are about to renaming `dateUpdated` to `timestamp` to align with GSMA's specs but that will be done in another PR made by us.